.github/workflows/docs-label.yaml

```
@@ -0,0 +1,28 @@
+# opens an issue in https://github.com/dbt-labs/docs-internal when a pr labeled with "needs-docs" is merged.
+
+name: Open issues in docs-internal repo when a PR is merged
+run-name: "Create an issue in docs-internal for PR #${{ github.event.pull_request.number }}"
+
+on:
+  pull_request_target:
+    types: [closed]
```

This allows adding the label after a PR gets merged to trigger this workflow.  Useful when we forget to add it.
```suggestion
    types: [labeled, closed]
```